setup.py

```diff
@@ -46,7 +46,6 @@
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
         "Programming Language :: Python",
-        "Programming Language :: Python :: 3.6",
```

Comment:
`setup.cfg` still says py36. I'm not sure if we even need that file. (I ran `git grep 6` locally)

Reply:
Nor do I, but I've fixed it anyhow :)